UWM-RAR: Subscribe CTA, LB Styling, & Readme 577587430

blaquehat opened the pull request:
# Description:
This adds an initial version of a readme file for the theme. The biggest change is an update to the Subscribe CTA code. Change code to take advantage of AlpineJS to fix an issue noticed on the test site. Add styling to Layout Builder Form Elements. Style System messages that appear in the theme.

## Subscribe CTA:
The changes add a page-level variable to track if cta is to be displayed or not. On window load, it checks if the cookie is set. If set it keeps the cta hidden. if not set it set variable to true and removes the hidden class from cta (this was added to reduce flicker). If cta is shown and x button is clicked, then the variable is set back to false and cta is applied with hidden class.

## Layout Builder Styling:
This changes the styling of Layout Builder form elements when they are shown in the Theme. Added styling to discard changes for Nodes.

## System Messages:
Add styling to system messages to distinguish between warning, errors, and regular status.

sodacrackers commented:
I've loaded this branch and configurations. I'm not seeing bugs with jump-to-top. Could additional form work be required? I see new Layout Builder styles on Discard Changes but not Revert to Defaults; on Discard Changes one button is 15% wide, one is full width.

blaquehat commented:
> I've loaded this branch and configurations. I'm not seeing bugs with jump-to-top. 

I did no work with Jump to Top, it was with Subscribe CTA.

blaquehat commented:
Update:
- Fixed the Revert section.
- Simplified the Form Alter Preprocesses for the other LB forms.
- Added Span to System Messages for text flow.
